src/installing-workflow/index.md

```diff
@@ -92,7 +92,8 @@ Depending on the order in which the Workflow components initialize, some pods ma
 installation: if a component's dependencies are not yet available, that component will exit and Kubernetes will
 automatically restart it.
 
-Here, you can see that controller, builder and registry all took a few loops before there were able to start:
+Here, it can be seen that the controller, builder and registry all took a few loops before there were able to start:
```

Comment:
I know you didnt change it, but you touched the line...

"there" should be "they"